src/routes/search/containers/SearchPage.js

```diff
@@ -45,7 +45,7 @@ export class SearchPage extends React.Component<PropsType, LocalStateType> {
     // find all categories whose contents but not titles include the filter text and sort them lexicographically
     const categoriesWithContent = categories.toArray()
       .filter(category => !normalizeSearchString(category.title).includes(filterText))
-      .filter(category => normalizeSearchString(category.content).includes(filterText))
+      .filter(category => normalizeSearchString(category.content).replace(/(<([^>]+)>)/ig, '').includes(filterText))
```

Comment:
Please add a comment why this is necessary and what kind of regex this is. Is this used to strip html tags?